### PR TITLE
Respect static list of allowed namespaces

### LINKED ISSUE #1634: Cluster accessible namespaces does not have any effect
**Describe the bug**

If user specifies "Accessible Namespaces" via cluster settings it does not have any effect what's displayed in Cluster dashboard.

**Expected behavior**

Cluster should show only specified namespaces.

### PR DESCRIPTION
**Pull request by jakolehm**
Fixes #1634

### CODE REVIEW

**Review by nevalla — APPROVED** [review]

File: src/renderer/components/+namespaces/namespace.store.ts [hunk 1]
```diff
@@ -41,6 +42,17 @@ export class NamespaceStore extends KubeObjectStore<Namespace> {
     };
   }
 
+  subscribe(apis = [this.api]) {
+    const { allowedNamespaces } = getHostedCluster();
+
+    // if user has given static list of namespaces let's not start watches because watch adds stuff that's not wanted
+    if (allowedNamespaces.length > 0) {
+      return () => { return; };
```
Comment (panuhorsmalahti) [inline src/renderer/components/+namespaces/namespace.store.ts:50]: Why "return;"?

Comment (Nokel81) [reply src/renderer/components/+namespaces/namespace.store.ts:50]: We have a lint against empty functions.

Comment (Nokel81) [inline src/renderer/components/+namespaces/namespace.store.ts:50]:
```suggestion
      return noop;
```

**Review by Nokel81 — APPROVED** [review]
Looks good, though I think we should mark that empty function explicitly as a desired noop

**Pull request merged**